scripts/test/Muon/elemental_analysis/periodic_combo_test.py

```diff
@@ -18,13 +18,15 @@
 @start_qapplication
 class PeriodicComboTest(unittest.TestCase):
     @mock.patch('Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table.QtWidgets.QComboBox.insertItem')
-    def test_that_init_defaults_to_all_elements_in_periodic_table(self, mock_insert_item):
+    def test_that_init_defaults_to_all_elements_in_periodic_table_with_details(self, mock_insert_item):
```

Comment:
can we make the 109 a constant at the top of the file with a sensible name? Then if we add more elements we can easily update it everywhere